Penteract/Assets/Scripts/Character.cpp

```diff
@@ -1,12 +1,19 @@
 #include "Character.h"
 
 void Character::GetHit(float damage_) {
+	bool wasAlive = lifePoints > 0.0f;
```

Comment:
What is the purpose of this bool?

Reply:
Modified to this commento:
	//We assume that the character is always alive when this method gets called, so no need to check if character was alive before taking lives